tasks/install_shell.sh

```diff
@@ -119,7 +129,11 @@ else
   if [ "$nightly" = true ]; then
     yum_source='https://artifactory.delivery.puppetlabs.net:443/artifactory/internal_nightly__local/yum'
   else
-    yum_source='http://yum.puppet.com'
+    yum_source='https://yum-puppetcore.puppet.com/public'
```

Comment:
Rather than change the default, we could introduce a new `puppetcore8` collection so that it's possible to incrementally move to puppetcore. This would complicated a few things as the old and new repos have different GPG keys and in the future could have diverging release package conventions.